README.md

```diff
@@ -2,25 +2,38 @@
 Created on: Jan 5, 2023
 Author: Chris Hanes for Good Filling LLC.
 
-PURPOSE: This library allows us to use the DRV8251 with our ESP32-S2. 
+PURPOSE: This library allows us to use the DRV8251 with the ESP32-S2. 
 
 DISCLAIMER: This is a work in progress. Changes are frequent and not guaranteed to work with your specific scenario. Review code carefully before using. Use at your own discretion
 
-Publishing Steps
-- Make changes to Motor.cpp and Motor.h
-- Test changes
-- ZIP the project
-- Drop the zipped project into the Publish directory
-- Push to Github
+## Usage Instructions
+This library was written specifically for the DRV8251, but it may work with other motor drivers, as well. Refer to the documentation provided by Texas Instruments for details on the DRV8251 (https://www.ti.com/lit/ds/symlink/drv8251.pdf) and to adapt this library to the specifics of your motor driver.
 
-Consumption Steps
-- Add a library at <>
-- Import as any other library
+Initialize the motor driver as follows:
+``` Motor motor = Motor(IN1, IN2, PWM Channel 1, PWM Channel 2); ```
```

Comment:
For initialization, also include that the motor driver doesn't require PWM objects:

Initialize the motor driver as follows:
Motor motor = Motor(IN1, IN2, int Channel1, int Channel2);
where Channel1 and Channel2 are integers corresponding to the PWM channel you want to attach (ESP32s have 16 independent PWM channels labeled 0-15)
IN1 and IN2 are the pins that are used to interface with a motor driver (TI's DRV8251) and the names line up with the pin-in from this chip